packages/thorium-deb/thorium-deb.pacscript

```diff
@@ -1,14 +1,31 @@
 name="thorium-deb"
 gives="thorium-browser"
-pkgver="117.0.5938.157"
-arch=('arm64' 'amd64')
-if [[ ${CARCH} == "arm64" ]]; then
-  url="https://github.com/Alex313031/Thorium-Raspi/releases/download/M${pkgver}/${gives}_${pkgver}_${CARCH}.deb"
-  hash="526c851d042702e5648ae959d60f7956c68473481bc5e507cdb1db7f0e9dac14"
-else
-  url="https://github.com/Alex313031/thorium/releases/download/M${pkgver}/${gives}_${pkgver}_${CARCH}.deb"
-  hash="9ae34160febcdf63e63f47adf4449a46977f048c18666c1d90786c30d04b404e"
-fi
+pkgver="119.0.6045.214"
+homepage="https://thorium.rocks"
+arch=('arm64' 'amd64' 'i386')
+case "${CARCH}" in
+  arm64)
+    url="https://github.com/Alex313031/Thorium-Raspi/releases/download/M${pkgver}/${gives}_${pkgver}_${CARCH}.deb"
+    hash="811dded7557b0ca9a93c3a50e5ce6658f2d185b9858443a76d359485598856b5"
+    ;;
+  i386)
+    url="https://github.com/Alex313031/Thorium-Other/releases/download/${pkgver}-2/${gives}_${pkgver}_${CARCH}.deb"
+    hash="ee5443c8547bb1828e7a105fb5f1ca4eaa43d5a16fd683a37bb1b2e9f57db21c"
+    ;;
+  amd64)
+    if grep -q avx2 /proc/cpuinfo; then
```

Comment:
What's avx?

Reply:
A CPU feature, google it. Thorium and Mercury provide specialized builds with it

Reply:
I don't like grepping in a pacscript, I *really* want all pacscripts to be simple control flow with no external commands.

Reply:
Again, if you have a better way I'm all ears. Pacstall does not have a way to specify compatible hardware features

Reply:
Then we should make one! Should we discuss on discord/matrix?